arm-templates/azuredeploy-ctm.json

```diff
@@ -75,601 +75,609 @@
       }
     },
     "isSecondary": {
-      "type":"bool",
+      "type": "bool",
       "defaultValue": false
     }
   },
-  "variables": {    
+  "variables": {
     // Duplicated resources
```

Comment:
this is illegal in json, i believe

Reply:
same for all similar comments

Reply:
its true but arm template is letting this slide